neps/nep-0508.md

```diff
@@ -106,116 +106,93 @@ This was implemented ahead of this NEP and the fixed shards feature was **remove
-* TBD
+* Required space to maintain state (either in memory or in persistent disk) will be smaller. This is useful for in-memory tries.
+* State sync overhead will be smaller with smaller state size.
 
 ### Neutral
 
-* Number of shards is expected to increase.
+* Number of shards would increase.
 * Underlying trie structure and data structure are not going to change.
-* Resharding will create dependency on flat state snapshots. 
+* Resharding will create dependency on flat state snapshots.
 
 ### Negative
 
-* The resharding process is still not fully automated. Analyzing shard data, determining the split boundary, and triggering an actual shard split all need to be manually curated by a person.
-* During resharding, a node is expected to do more work as it will first need to copy a lot of data around the then will have to apply changes twice (for the current shard and the future shard).
+* The resharding process, as of now, is not fully automated. Analyzing shard data, determining the split boundary, and triggering an actual shard split all need to be manually curated and tracked.
+* During resharding, a node is expected to require more resources as it will first need to copy state data from the parent shard to the child shard, the then will have to apply trie and flat state changes twice, once for the parent shard and once for the child shards.
 * Increased potential for apps and tools to break without proper shard layout change handling.
 
 ### Backwards Compatibility
 
 We do not expect anything to break with this change. Yet, shard splitting can introduce additional complexity on replayability. For instance, as target shard of a receipt and belonging shard of an account can change with shard splitting, shard splitting must be replayed along with transactions at the exact epoch boundary.
 
-## Unresolved Issues (Optional)
-
-[Explain any issues that warrant further discussion. Considerations
-
-* What parts of the design do you expect to resolve through the NEP process before this gets merged?
-* What parts of the design do you expect to resolve through the implementation of this feature before stabilization?
-* What related issues do you consider out of scope for this NEP that could be addressed in the future independently of the solution that comes out of this NEP?]
-
-## Changelog
-
-[The changelog section provides historical context for how the NEP developed over time. Initial NEP submission should start with version 1.0.0, and all subsequent NEP extensions must follow [Semantic Versioning](https://semver.org/). Every version should have the benefits and concerns raised during the review. The author does not need to fill out this section for the initial draft. Instead, the assigned reviewers (Subject Matter Experts) should create the first version during the first technical review. After the final public call, the author should then finalize the last version of the decision context.]
-
-### 1.0.0 - Initial Version
```

Comment:
I thought this should stay here and be filled in when reviewed. No clue if that is how it works actually.

Reply:
Oh right, sorry, I just took a look at some other NEPs and they didn't have this, so maybe I thought it shouldn't be there. My bad, will add in